nodeJS/testing_express/testing_routes_and_controllers.md

```diff
@@ -58,11 +58,14 @@ These two files, `app.js` and `index.js` simply define a couple of routes and th
 
 To facilitate actually testing these routes we're going to use a library called [SuperTest](https://github.com/visionmedia/supertest), so go ahead and `npm install supertest --save-dev` and while it's installing take a few minutes to look through the readme on their git repo (linked above).
 
+As well, you will need to install Jest and create a new npm script which runs Jest. If you've forgotten how to do this then this may be a good time to take a look [at the previous Javascript testing lessons](https://www.theodinproject.com/lessons/node-path-javascript-testing-basics).
+
 In the examples below we're going to use SuperTest inside of a Jest style describe/test block, but the syntax and use of these are common among most testing libraries, so the concepts should be easily replicated in Mocha with Chai or Jasmine or Tape or whatever testing library you prefer.
 
 Here's our test file:
 
 ~~~javascript
+//// index.test.js
```

Comment:
Was this intended to be left here? Just wanna make sure.

Reply:
I did put that there intentionally for clarity but feel free to remove it if you'd like. I saw that style of commenting the file that you are in code blocks from other Odin lessons.